lib/noble.js

```diff
@@ -33,23 +33,31 @@ module.exports = {
   },
   privateKeyTweakAdd: function (privateKey, tweak) {
     try {
-      const res = secp256k1.privateAdd(privateKey, tweak)
+      const res = bitcoinerlab.privateAdd(privateKey, tweak)
       if (!res) return 1
       privateKey.set(res)
     } catch (e) {
       return 1
     }
     return 0
   },
-  privateKeyTweakMul: function (privateKey, tweak) {
-    return elliptic.privateKeyTweakMul(privateKey, tweak)
+  privateKeyTweakMul: function (seckey, tweak) {
+    try {
+      const keyNum = noble.utils._normalizePrivateKey(seckey)
+      const tweakNum = noble.utils._normalizePrivateKey(tweak)
+      const res = noble.utils.mod(keyNum * tweakNum, noble.CURVE.n)
+      seckey.set(noble.utils._bigintTo32Bytes(res))
+    } catch (e) {
+      return 1
```

Comment:
let's at least log it, esp if we don't expect this to ever happen
```suggestion
    } catch (e) {
      console.error('privateKeyTweakMul failed', e) 
      return 1
```

Reply:
@mvayngrib this is thrown from `lib/index.js`. Why do we need to log it?

Reply:
yeah, but we throw a new error from there, and lose the underlying reason from here. Not a blocker